pyte/streams.py

```diff
@@ -129,7 +129,7 @@ class Stream(object):
 
     #: A regular expression pattern matching everything what can be
     #: considered plain text.
-    _special = set([ctrl.ESC, ctrl.CSI, ctrl.NUL, ctrl.DEL, ctrl.OSC, ctrl.ST])
```

Comment:
On second thought, both `ST` and `BEL` are terminators, the regex wouldn't match the text containing them anyway, right?